### PR TITLE
Feat 1180 - Sortable icon on hover

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

Implements only showing "sortable" icon on hover per https://github.com/grommet/hpe-design-system/issues/1180

[Preview](https://deploy-preview-1218--keen-mayer-a86c8b.netlify.app/components/table)

#### Where should the reviewer start?

aries.js

#### What testing has been done on this PR?

[DataTable page](https://deploy-preview-1218--keen-mayer-a86c8b.netlify.app/components/table)

#### How should this be manually tested?

[DataTable page](https://deploy-preview-1218--keen-mayer-a86c8b.netlify.app/components/table)

#### What are the relevant issues?

https://github.com/grommet/hpe-design-system/issues/1180

#### Is this change backwards compatible or is it a breaking change?

Backwards compatible

### CODE REVIEW

**Review by vavalos5 — APPROVED** [review]
Looks good to me! Thanks Matt! 😃

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -32,6 +30,23 @@ export const aries = deepMerge(hpe, {
...
+          ${sortable &&
+            sort &&
+            sort.property !== column &&
+            `
+              svg {
+                opacity: 0%;
+              }
+              :hover {
+                svg {
+                  opacity: 50%;
```
Comment (taysea) [inline aries-site/src/themes/aries.js:44]:
```suggestion
                  opacity: 100%;
```
Designs seem to be calling for full opacity of `text-strong`: https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A13951

File: aries-site/src/themes/aries.js [hunk 2]
```diff
@@ -32,6 +30,23 @@ export const aries = deepMerge(hpe, {
...
       border: { side: 'bottom' },
       color: 'text-strong',
+      extend: ({ column, sort, sortable }) => {
+        return `
+          ${sortable &&
+            sort &&
+            sort.property !== column &&
+            `
+              svg {
+                opacity: 0%;
```
Comment (taysea) [inline aries-site/src/themes/aries.js:40]:
```suggestion
                opacity: 0;
```

**Review by taysea — COMMENTED** [review]
Just one small comment

File: aries-site/src/themes/aries.js (hunk 1 above)
Comment (halocline) [reply aries-site/src/themes/aries.js]: Done. Thanks.

File: aries-site/src/themes/aries.js (hunk 2 above)
Comment (halocline) [reply aries-site/src/themes/aries.js]: Done. Thanks.

**Review by taysea — APPROVED** [review]
LGTM. It appears that `text-strong` is not propagating to the icon color. Can you link the separate grommet issue you file to fix this so we can track separately? @halocline

**Comment by halocline** [comment]
> LGTM. It appears that `text-strong` is not propagating to the icon color and we are falling back to `text`. Can you link the separate grommet issue you file to fix this so we can track separately? @halocline

https://github.com/grommet/grommet/issues/4677

**Pull request merged**